.python-version

```diff
@@ -0,0 +1 @@
+3.10.4
```

Comment:
Hey @rposborne, thank you for the shoutout!

I was using this file on my branch because I use pyenv. This file need not be checked in.